apps/testing-javascript/package.json

```diff
@@ -61,6 +61,7 @@
     "sanity": "^3.0.0",
     "sanity-plugin-cloudinary": "^1.0.1",
     "styled-components": "^5.2",
+    "superjson": "^1.12.2",
     "uuid": "^8.3.2",
     "zod": "^3.20.2"
```

Comment:
zod and superjson should be consistent versions across all apps/packages too I think